tests/foreman/api/test_computeresource_vmware.py

```diff
@@ -0,0 +1,112 @@
+"""
+:Requirement: Computeresource Vmware
+
+:CaseComponent: ComputeResources-VMWare
+
+:Team: Rocket
+
+:CaseImportance: High
+
+:CaseAutomation: Automated
+
+"""
+
+from fauxfactory import gen_string
+import pytest
+from wait_for import wait_for
+
+from robottelo.config import settings
+
+
+@pytest.mark.e2e
+@pytest.mark.on_premises_provisioning
+@pytest.mark.parametrize('setting_update', ['destroy_vm_on_host_delete=True'], indirect=True)
+@pytest.mark.parametrize('vmware', ['vmware7', 'vmware8'], indirect=True)
+@pytest.mark.parametrize('pxe_loader', ['bios', 'uefi'], indirect=True)
+@pytest.mark.parametrize('provision_method', ['build', 'bootdisk'])
+@pytest.mark.rhel_ver_match('[^6]')
```

Comment:
Can we split execution of tests with EL8 and EL9 on API and CLI?

Reply:
This would be a good idea and would also reduce the provisioning pipeline execution time.